testing/grpc-java/APKBUILD

```diff
@@ -0,0 +1,25 @@
+# Contributor: wener <wenermail@gmail.com>
+# Maintainer: wener <wenermail@gmail.com>
+pkgname=grpc-java
+pkgver=1.10.0
+pkgrel=0
+pkgdesc="The Java gRPC implementation. HTTP/2 based RPC"
+url="https://github.com/grpc/grpc-java"
+arch="all"
```

Comment:
Maybe java is `noarch`?

Reply:
No, compiler contain cpp
https://github.com/grpc/grpc-java/blob/master/compiler/src/java_plugin/cpp/java_plugin.cpp